terralab/utils.py

```diff
@@ -27,10 +27,19 @@ def wrapper(*args: str, **kwargs: str) -> Any:
         try:
             return func(*args, **kwargs)
         except ApiException as e:
+            message = "[no message body]"
             if e.body is not None:
-                formatted_message = f"API call failed with status code {e.status} ({e.reason}): {json.loads(e.body)['message']}"
-            else:
-                formatted_message = f"API call failed with status code {e.status} ({e.reason}): [no message body]"
+                message = json.loads(e.body)["message"]
+            if e.status == 401 and message == "User not found":
```

Comment:
just curious, are there 401s that have different messages that we're worried about here?

Reply:
yea im wondering if we want to return this message on all 401s or ones with this specific message.  I would prob lean to all 401s and if people see 401s that arent this issue we can adjust (as opposed to pre specifying) but i would imagine this is our only 401 we'll see so im good either way :)